src/Components/Button/Button.js

```diff
@@ -48,7 +48,7 @@ Button.propTypes = {
   label: PropTypes.string,
   color: PropTypes.string,
   uniqueBtn: PropTypes.string,
-  icon: PropTypes.arrayOf(PropTypes.string),
+  icon: PropTypes.oneOfType([PropTypes.string, PropTypes.array]),
```

Comment:
Hmm, this looks quite strange and potentially error prone. Arrays are complex objects and someone may iterate on it, get values etc in the functions in this component. How's that possible that both arrays and strings might be passed here?